### PR TITLE
Fixing items on 4.1

### PR DESCRIPTION
**Pull request by ablqk**

### CODE REVIEW

File: chapter-04/1_3_isolate_validate/gordle/game_internal_test.go [hunk 1]
```diff
@@ -81,9 +69,9 @@ func TestGameValidateAttempt(t *testing.T) {
 
 	for name, tc := range tt {
 		t.Run(name, func(t *testing.T) {
-			g := &Game{}
+			g := New(nil)
 
-			err := g.validateAttempt(tc.word)
+			err := g.validateGuess(tc.word)
```
Comment (floppyzedolfin) [inline chapter-04/1_3_isolate_validate/gordle/game_internal_test.go:74]: rename to `guess` ?

File: chapter-04/1_4_check_for_victory/gordle/game.go [hunk 2]
```diff
@@ -58,30 +60,31 @@ func (g Game) ask() []rune {
...
+		return fmt.Errorf("expected %d, got %d, %w", len(g.solution), len(guess), errInvalidWordLength)
 	}
 
 	return nil
 }
 
 // splitToCharacters is a naive implementation to turn a string into a list of characters.
-func splitToCharacters(solution string) []rune {
-	return []rune(solution)
+func splitToUppercaseCharacters(solution string) []rune {
```
Comment (floppyzedolfin) [inline chapter-04/1_4_check_for_victory/gordle/game.go:88]: giphy

**Review by floppyzedolfin — APPROVED** [review]

File: chapter-04/2_hints/go.mod [hunk 3]
```diff
@@ -1,3 +1,5 @@
-module github.com/ablqk/tiny-go-projects/chapter-04/2_feedback
+module github.com/ablqk/tiny-go-projects/chapter-04/2_hints
```
Comment (floppyzedolfin) [inline chapter-04/2_hints/go.mod:1]: nice catch

giphy

**Review by floppyzedolfin — APPROVED** [review]

File: chapter-04/1_3_isolate_validate/gordle/game_internal_test.go [hunk 4]
```diff
@@ -81,9 +69,9 @@ func TestGameValidateAttempt(t *testing.T) {
 
 	for name, tc := range tt {
 		t.Run(name, func(t *testing.T) {
-			g := &Game{}
+			g := New(nil)
```
[the remark this answers is not in the document]
Comment (ablqk) [reply chapter-04/1_3_isolate_validate/gordle/game_internal_test.go:72]: As agreed today, let's keep it a New

**Pull request merged**